contracts/ICO/EasyMineIco.sol

```diff
@@ -96,20 +102,26 @@ contract EasyMineIco {
     }
   }
 
-  function setup(address _easyMineToken, address _sys)
+  function setup(address _easyMineToken, address _sys, address _resAdr, uint256 _startDelay, uint256 _icoDuration)
     public
     isOwner
     atStage(Stages.Deployed)
   {
     require(_easyMineToken != 0x0);
     require(_sys != 0x0);
+    require(_resAdr != 0x0);
+    require(_startDelay != 0x0);
```

Comment:
let's change those two (`_startDelay` and `_icoDuration`) to `require( ... > 0);`